jennifer_choi/HappiTails/lib/seeds.rb

```diff
@@ -0,0 +1,24 @@
+@HappiTails = Shelter.new("HappiTails")
```

Comment:
This is one way to make the shelter kind of global, but might run into namespace issues. If this is to be a one-shelter app you can make globals like: `$HAPPITAILS`

anther way would be to put the seed stuff into a method that returns the shelter, then in main, set a variable to calling that method.